@types/contentful-html-rich-text-converter/index.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module 'contentful-html-rich-text-converter' {
+  function parseHtml(html: string): {};
```

Comment:
this would need a better look, I only added a type because TS was complaning about `contentful-html-rich-text-converter` not having a type

Reply:
this is fine, it doesn't come with it's own types and I take it you have looked in definitely-typed too?

I would just do
```suggestion
  function parseHtml(html: string): unknown;
```

I don't think `{}` is a good type, if you want to make it an object you could use some sort of Record type, the question is what do you need for it to work with the client lib.

Reply:
btw maybe you should move it into the @types dir in the root?